rpc/documentation/api.md

```diff
@@ -1615,6 +1681,53 @@ wallet's relevant transactions contained therein.
 
 ___
 
+#### `TicketDetails`
+
+The `TicketDetails` message is included in responses to report transactions
+relevant to the wallet.  The message includes details such as which previous
+wallet inputs are spent by this transaction, whether each output is controlled
+by the wallet or not, the total fee (if calculable), and the earlist time the
+transaction was seen.
+
+- `bytes hash`: The hash of the given ticket.
+
+- `bytes spender_hash`: The hash of the spender (if applicable, otherwise empty).
+
+- `int64 ticket_age`: If unspent the blocks since being mined.  Otherwise,
+   the number of blocks from when it was mined to when it was spent.
+
+- `int64 ticket_price`: The value of the first output from the ticket transaction.
+  Due to the way tickets are constructed, we know that this is always the 'price'
+  of any given ticket.
+
+- `int64 ticket_cost`: The total cost of the wallet to purchase the ticket.  Typically,
+  this would just be the ticket fee.
+
+- `int64 spender_return`: The total return from the spender transaction.  If a vote,
+  this number should be positive (PoS vote subsidy - poolfee).  If a revocation, 
+  this would be negative (minus txfee and poolfee). 
+
+- `TicketStatus ticket_status`: The observed status of the given ticket.
+
+  **Nested enum:** `TicketStatus`
```

Comment:
The ordering of these seems odd.  I would start with unmined, then immature, then live, followed by the rest.